### PR TITLE
LRQA-68303 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-68303

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:relevant

**Comment by melodywu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/UsersAdmin.testcase [hunk 1]
```diff
@@ -811,6 +811,37 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for can add comment for user."
+	@priority = "2"
+	test CanAddComment {
+		ApplicationsMenu.gotoPortlet(
+			category = "Users",
+			panel = "Control Panel",
+			portlet = "Users and Organizations");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/UsersAdmin.testcase:820]: Pull looks good! Just a minor change, can we start using` User.openUsersAdmin();` to make our tests run faster? I think a test or two that will navigate to Users Admin via UI is good enough. Same change for line 831. Thanks!

**Comment by patriciajeanperez** [comment]
ci:close

**Pull request closed without merging**